lib/transbank/common/request_service.ts

```diff
@@ -35,7 +35,8 @@ const RequestService = {
       })
       .catch((error) => {
         let response = error.response;
-        throw new TransbankError(error, response.data.error_message);
+        let msg = response?.data?.error_message;
```

Comment:
Shouldn't we give out a default message if none is found ?

Reply:
I agree with @ffiebig this kind of error handling will get you a TrasnbankError(error, undefined) thats a little week.